.github/workflows/check-reactions-on-pr-and-issues.yml

```
@@ -0,0 +1,55 @@
+name: Check Reactions on Pr and Issue
+on:
+  schedule:
+    - cron: "0 0 * * 0" # Run at midnight on Sundays
```

Not a fan of midnight. A random time would be a lot better.
```suggestion
    - cron: "0 13 * * 0" # Run at 1 PM on Sundays
```